packages/cli/package.json

```diff
@@ -33,6 +33,7 @@
     "jsonschema": "^1.4.0",
     "node-machine-id": "^1.1.12",
     "open": "^8.4.0",
+    "os-name": "^4.0.1",
```

Comment:
Meh, can we do without an additional dependency ?

With Snyk/vulnerabilities analysis and what not, adding new dependency need to be for critical reasons /feature, and not "nice to have".

Is this absolutely needed ?

Reply:
Not critical. As long as we have the platform and the version, we can always find back the OS name